CODEOWNERS

```diff
@@ -0,0 +1,28 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# https://github.blog/2017-07-06-introducing-code-owners/
+
+# Assign code owners that will automatically get asked to review Pull Requests
+# The last matching pattern takes the most precedence.
+
+# These owners will be the default owners for everything in the repo.
+# Unless a later match takes precedence, they will be requested for
+# review when someone opens a pull request.
+* @enolfc @glarocca @gwarf
+
+# Operations
+content/*/operations/ @gwarf @paolini78
+
+# Providers
+content/*/providers/ @gwarf @paolini78
+
+# Users
+content/*/users/ @enolfc @glaroccca @andrea-manzi
+
+# DataHub
+content/*/users/datahub/ @andrea-manzi
+
+# FedCloud
```

Comment:
can you add me also to `content/*/providers/fedcloud` ?

Reply:
Great idea, done together with some other more specific doc, we will see if wildcards works as expected